go.mod

```diff
@@ -1,26 +1,23 @@
-module github.com/cyriltovena/logger
+module github.com/ViaQ/cluster-logging-load-client
 
-go 1.14
+go 1.16
 
 require (
+	github.com/bwplotka/bingo v0.4.3 // indirect
```

Comment:
This looks like a `go.mod` polluted by `bingo get`. The target aim for bingo is to have separate go.mod files per tool under `.bingo` without polluting this file.

Reply:
fixed
DONE